src/index.html

```diff
@@ -6,19 +6,22 @@
     <title>leaderboard</title>
   </head>
   <body>
-    <h1 class="page-title">leaderboard</h1>
+    <h1 class="page-title">leaderboard <span class="smaller">RIZZY Games</span></h1>
+    
```

Comment:
## Status: ✔️ Project Approved ✔️ 
----

Hello @Elerqsousy  🔥 

> # Excellent work on your project, it looks very professional and it's completed. Congrats on the good work!

![surprised](https://user-images.githubusercontent.com/60631456/122959988-c881fc00-d348-11eb-988a-a5aa782f4777.gif)

Now you can celebrate! 🥇 🚀 👍 💯 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️
`There's nothing left to say but time to merge! 😊`

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._